src/domains/job/search/components/WriteJobSearchPostingForm.jsx

```
@@ -12,39 +12,48 @@ import useAuthStore from '../../../../store/login/useAuthStore'
 import useIsMobile from '../../../../hooks/header/useIsMobile'
 import useFreeDraftStore from '../../../../store/mypage/useFreeDraftStore'
 
-const WriteJobSearchPostingForm = ({ onSubmit, editorRef }) => {
+const WriteJobSearchPostingForm = ({ onSubmit, editorRef, defaultValues }) => {
```

ESLint

[error] 'onSubmit' is missing in props validation

(react/prop-types)